webapp/css/participants.css

```diff
@@ -15,7 +14,37 @@ body{
 #header-banner   {
     border: 1px solid #ccc;
     background-color: whitesmoke;
-    width: 100%; 
+    width: 105%; 
```

Comment:
as someone unfamiliar with CSS that much, why would you want the width to be 105%?

Reply:
Fixed. 

I noticed a gap in the participants page that was not in the search page, and the search page set its width to 105% to reach the edge of the page. This gap is actually due to the body section automatically adding an 8px margin, so I changed the body margin to 0px and removed the confusing code.